.travis.yml

```diff
@@ -28,3 +28,5 @@ install:
 script:
   - python -m unittest discover tests -v
```

Comment:
Why `coverage` is not used instead of `python`. Similar to #172

Reply:
I'm not sure I'm folllowing, @devkapilbansal . Isn't the next line (line 30) means we're using coverage?